### ISSUE TITLE
Move tooling related docs from arcade to dnceng

### ISSUE THREAD

**Issue by dkurepa** [dotnet/arcade#13483]
Currently, most of our docs are stuck in arcade, even tho there's lots of them that don't belong there.
We should go through them and move everything to the new dnceng repo, if we think they belong there

**Comment by garath** [comment]
Say more about "don't belong there", what are you seeing?

**Comment by missymessa** [comment]
This stuff should be moved to dotnet/dnceng: https://github.com/dotnet/arcade/tree/main/Documentation/TeamProcess

(I called out Rollout Scorecards being added to the dnceng repo, and the epic process in the wiki calls out putting one-pagers in the AzDO wiki).

**Comment by missymessa** [comment]
Some of this should also move, but I think some of this might be related to features in Arcade: https://github.com/dotnet/arcade/tree/main/Documentation/Project-Docs

**Comment by dkurepa** [comment]
Yup, as Missy pointed out, things related to our internal processes.
